eng/common/scripts/Package-Properties.ps1

```diff
@@ -1,21 +1,20 @@
 # Helper functions for retireving useful information from azure-sdk-for-* repo
-# Example Use : Import-Module .\eng\common\scripts\modules
 class PackageProps
 {
-    [string]$pkgName
-    [string]$pkgVersion
-    [string]$pkgDirectoryPath
-    [string]$pkgServiceName
-    [string]$pkgReadMePath
-    [string]$pkgChangeLogPath
-    [string]$pkgGroup
-
-    PackageProps([string]$pkgName,[string]$pkgVersion,[string]$pkgDirectoryPath,[string]$pkgServiceName)
+    [string]$Name
+    [string]$Version
+    [string]$DirectoryPath
+    [string]$ServiceName
```

Comment:
I know in a lot of places we call this ServiceDirectory and I suspect we should be consistent here so folks don't mistake it for the nice display name for the service.

Reply:
We probably want to do that consistently through these scripts so it matches all the other places in our pipelines that call it ServiceDirectory.